packages/ts/src/components/donut/index.ts

```diff
@@ -38,6 +38,9 @@ export const [
   DONUT_HALF_ANGLE_RANGE_LEFT,
 ] = DONUT_HALF_ANGLE_RANGES
 
+const DONUT_HALF_LABEL_OFFSET_Y_DEFAULT = 20
```

Comment:
We've already briefly discussed that instead of having these as constants, we should determine them dynamically based on the label size.

Reply:
Indeed. We could try based on the font size, but that won't work if the text is wrapped, which I see is an option available called `centralSubLabelWrap`.

Reply:
Good point, you can use `estimateTextSize` to get the width and hight.

Reply:
Done